src/Card.js

```diff
@@ -1,12 +1,15 @@
 import React from 'react';
 import {Link} from 'react-router-dom';
 import HeapAvatars from './components/HeapAvatars/HeapAvatars';
+import {connect} from 'react-redux';
+import {bindActionCreators} from '../node_modules/redux';
```

Comment:
don't use `node_modules` in import, just `'redux` in this case and read ^^^ comment